### DISCUSSION TITLE
MQTT problems

### DISCUSSION THREAD

**Discussion by dheuts90** [arendst/Tasmota#17286] [Q&A]
Good morning,

I am having troubles with MQTT on my Tasmota devices via MQTT (From Domoticz) since last friday. Also without any MQTT errors or notifications in the console.

I have seen that when I use the webinterface of Tasmota to switch the lights on and off (toggle), it works very quick. Also the switch in Domoticz follows directly, so MQTT is working partially. It's working from Tasmota to Domoticz, but not the other way around.

When I switch in Domoticz, I see the Domoticz/out MQTT message via MQTT Explorer directly. But the Tasmota switch is not responding, or only after minutes.

Yesterday I did some tests, and have setup another MQTT server. So now I have 2 MQTT brokers connected to my Domoticz, and I have setup the new broker (.246) to some Tasmota devices only. They are responding very quickly now again, while the other Tasmota devices, connected to the other MQTT broker (.243) are still responding very slow.

Any idea what could be the reason for this? Thanks in advance

**Comment by barbudor** [comment]
Hi @dheuts90
As I understand the problem doesn't follow any tasmotized device but seems to be attached to 1 of the broker

Are both RPi setup the same? Same model? 
Connected to the same AP or through eth? 
Have you check the CPU load on both? Could the slow one be overloaded?
